tests/components/component-playground/selected-fixture-and-editor/render/children.js

```diff
@@ -0,0 +1,23 @@
+var FIXTURE = 'selected-fixture-and-editor';
+
+describe(`ComponentPlayground (${FIXTURE}) Render Children`, function() {
+  var loadChild = require('react-component-tree').loadChild,
+      render = require('tests/lib/render-component.js'),
+      stubLoadChild = require('tests/setup/stub-load-child.js'),
+      fixture = require(`fixtures/component-playground/${FIXTURE}.js`);
+
+  var component,
+      $component,
+      container,
+      fixture;
```

Comment:
I don't think we need this kind of indentation here. They should fit on one line.

Also isn't fixture declared already a few lines above?